package.json

```diff
@@ -7,7 +7,7 @@
     "concurrently": "^9.0.1"
   },
   "dependencies": {
-    "react-router-dom": "^6.26.2",
-    "mongoose": "^8.6.3"
+    "mongoose": "^8.6.3",
+    "react-router-dom": "^6.27.0"
```

Comment:
do not install these dependencies at the route, please, this is the second time it happens